File: app/src/main/java/com/example/user/mathmaniac/MainMenuActivity.java
package com.example.user.mathmaniac;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import com.google.android.gms.ads.MobileAds;

public class MainMenuActivity extends AppCompatActivity {
    private ImageView btnPlay;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_menu);
        MobileAds.initialize(this, "ca-app-pub-5618073725839874~4344928149");
        addControls();
        addEvents();

    }

    private void addEvents() {
        btnPlay.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainMenuActivity.this, PlayActivity.class);
                startActivity(intent);
            }
        });
    }

    private void addControls() {
        btnPlay = (ImageView) findViewById(R.id.btnPlay);
    }
}

File: app/src/main/java/com/example/user/mathmaniac/ui/MainMenuActivity.kt
package com.example.user.mathmaniac.ui

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.example.user.mathmaniac.R
import kotlinx.android.synthetic.main.activity_main_menu.*

class MainMenuActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_menu)

        btnPlay.setOnClickListener {
            val intent = Intent(this@MainMenuActivity, PlayActivity::class.java)
            startActivity(intent)
        }
    }

}
